Improve README.md

Panquesito7 opened the pull request:
Improved "README.md" (if there's something you don't like, feel free to change it).
And...found another **big** typo on "mod.conf". 😒

TumeniNodes commented:
I will just comment here, rather than as a review.
I would prefer to revert back to "works with 5.0.x" rather than saying "it may work with later releases"
Also, it seems you removed the section where I announced the date the doors were added, and the types of doors? I would prefer that left in as well.

Other than those 2 points, it all looks great, you do a nice job with README files, and thank you for the time and work, it is greatly appreciated.
I am so lazy, I really should go through all my mods and update the mod.conf files

Panquesito7 commented:
> Also, it seems you removed the section where I announced the date the doors were added, and the types of doors? I would prefer that left in as well.

Sorry for that! I've added it again. Please check if it is correct.

> I would prefer to revert back to "works with 5.0.x" rather than saying "it may work with later releases"

Fixed. Please check if it is correct.

> Other than those 2 points, it all looks great, you do a nice job with README files, and thank you for the time and work, it is greatly appreciated.

Thank you! 👍

TumeniNodes commented:
Thank you, it's perfect.
And no need to say "sorry"
